bedrock/mozorg/views.py

```diff
@@ -202,3 +203,14 @@ def about_view(request):
         template_name = 'mozorg/about.html'
 
     return l10n_utils.render(request, template_name)
+
+
+def moss_view(request):
+    locale = l10n_utils.get_locale(request)
+
+    if lang_file_is_active('mozorg/moss/index-092018', locale):
```

Comment:
Is this logic worth adding a test for?

Reply:
Perhaps, but I'm hopeful it will be localized quickly so the logic may be short-lived. It's only translated in Hindi and I don't know how they're going about handling updates.